woodwork/statistics_utils.py

```diff
@@ -106,6 +124,20 @@ def _get_describe_dict(dataframe, include=None, callback=None):
         values["physical_type"] = series.dtype
         values["logical_type"] = logical_type
         values["semantic_tags"] = semantic_tags
+
+        # Calculate extra detailed stats, if requested
+        if extra_stats:
+            if column.is_numeric:
+                values["histogram"] = _get_histogram_values(series, bins=bins)
+                if isinstance(column.logical_type, (Integer, IntegerNullable)):
+                    _range = range(int(values["min"]), int(values["max"]) + 1)
+                    if len(_range) <= bins:
+                        values["top_values"] = _get_top_values_numeric(series, _range)
+            elif column.is_categorical:
+                values["top_values"] = _get_top_values_categorical(series, top_x)
+            elif column.is_datetime:
+                values["recent_values"] = _get_recent_values(series, recent_x)
```

Comment:
I think we probably could do that, but I'd propose we hold off on doing that for now since that wasn't part of the initial implementation we are transferring over to Woodwork. Maybe we wait and see if we identify a need for that and implement at that time?